### ISSUE TITLE
yarn db:up (cascading) error with Node v12.14.0 and npm v6.13.4

### ISSUE THREAD

**Issue by thedavidprice** [hammerframework/example-blog#12]
From a fresh clone, ran `yarn install`, `cd api`, and `yarn db:up`.

**1) Initial Error Running from Fresh Clone**
Encountered this error: https://github.com/prisma/prisma2/issues/1162 Suggestion was to upgrade to Prisma2 preview 17+, which I did. 

**2) Error with api/prisma/schema.prisma config**
Upon update to Prisma2 preview017.1, I encountered an error with `api/prisma/schema.prisma` using env() in provider arg. Updated to `provider = "sqlite"`. 

See doc:
https://github.com/prisma/prisma2/blob/7f8b064e28402740aa29481fbc3aff6a8ef36b25/docs/prisma-schema-file.md#using-environment-variables 
>It is not possible to use environment variables for the provider argument in datasource and generator definitions

_note: I later came back and ran using Prisma preview018.2 and can confirm it works._

**3) Breaking change updating to Prisma2 preview016+**
See: https://github.com/prisma/prisma2/releases/tag/2.0.0-preview016
>Due to a major refactoring in Lift's migration engine, your migrations are likely going to break if you upgrade to preview016 with an error similar to this: Error parsing the migration steps: Error("unknown field 'name', expected 'model'", line: 1, column: 59).
>To get rid of this error, you'll need to manually delete the generated migrations folder from your file system and drop the _Migration table in your database.

I removed dev.db and migrations/

**4) binaryTarget error in schema.prisma**
Lastly, I ran into this error:
```
Error: Binary target linux-glibc-libssl1.0.2 doesn't exist anymore. Please use debian-openssl-1.0.x instead.
```

Updated schema.prisma `binaryTargets = ["darwin", "debian-openssl-1.0.x"]`

**🎉 Success! <whew>**
Confirm now working using the following:
- Node v12.14.0
- npm v6.13.4
- yarn v1.21.1
- prisma2 v2.0.0-preview018.2

**Comment by thedavidprice** [comment]
After completing the above, I encountered a separate error when running `yarn workspace api db:seed`

```
Invalid `photon.()` invocation in /Users/price/Repos/hammer-example-blog/api/prisma/seeds.js:37:25

Reason: Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(SqliteFailure(Error { code: Unknown, extended_code: 1 }, Some("no such table: dev.Tag"))
```

Seems like possible breaking change from using prisma2 v2.0.0-preview018.2 that's affecting shema.prisma config.

**Comment by cannikin** [comment]
Yeah, prisma2 is in pretty rapid development right now and it's been an adventure keeping up with the changes! We've been working on a tutorial and haven't updated the example-blog in a couple of weeks. We'll be doing that soon and hopefully that'll resolve these issues!

**Comment by thedavidprice** [comment]
Thanks, @cannikin Did this mostly for @mojombo to document the issue I was having. And I promise I was only trying to go through the tutorials and offer some feedback per his request. Then, well, basically this happened:
https://xkcd.com/349/

Excited about this project! Keep on keeping on.

**Comment by thedavidprice** [comment]
Confirming this is fixed as of update to Prisma2 2.0.0-preview019. Thanks!
